panel/viewable.py

```diff
@@ -264,6 +264,8 @@ def show(self, port=0, websocket_origin=None, threaded=False,
           A string title to give the Document (if served as an app)
         verbose: boolean (optional, default=True)
           Whether to print the address and port
+        show : boolean (optional, default=True)
+          Whether to open the server in a new browser tab on start
```

Comment:
Thanks!  I guess this is called "show" because it does what `bokeh serve --show` does.  But it's still very confusing to see `.show(show=False)`.  Is there any enthusiasm for renaming `show` here to `open` or `launch` or `browse`?